packages/dgt-utils/lib/utils/add-protocol-prefix.ts

```diff
@@ -12,6 +12,8 @@ export const addProtocolPrefix = async (uri: string): Promise<string> => {
 
   try {
 
+    if (uri.match('^https?://'))  return uri ;
```

Comment:
```suggestion
  if (uri.match('^https?://'))  return uri ;

  try {
```

Reply:
this would throw errors if `uri` is undefined, why does this need to be outside of the try/catch?

Reply:
uri is already typed as string. If any codepath could call this function with undefined the code should not build.